### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by aron-net**
[Project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md)
- Implement the design from the sneak peek wireframes.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+    <div class='user-card img'>
+      <h2>Name: <%= user.name %></h2>
+      <h3 class='user-container'>Number of posts:<%= user.posts_counter %></h3>
+       <button>
+          <a href='/users/<%= user.id %>'> User Details </a>
+       </button>
+    </div>
+  </div>
+<% end %>
+
```
Comment (cwaku) [inline app/views/users/index.html.erb]: [OPTIONAL]:
- [ ] I recommend below a better practice for displaying the users.
```ruby
   # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,14 @@
...
+     <div class='padd'>
+        <h5>Comments:<%= @post.comments_counter %></h5>
+        <h5>Likes:<%= @post.likes_counter %></h5>
+     </div>
+    <h4><%= @post.text %></h4>
+    <hr/>
+    <% @post.recent_comment.each do |comment| %>
+        <h5><%= comment.user.name %>: <%= comment.text %> </h5>
+    <% end %>
+</div>
```
Comment (cwaku) [inline app/views/posts/show.html.erb]: - [ ] Post show page should display the post with all comments listed as shown on the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=ABIFLA6OIXRW5GF52AJGR33BHIMGO). Kindly display all comments and not only the recent on the show page.

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @aron-helu 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
There are a few issues 🐛  that you still need to work 🛠️  on to go to the next project ⏭️, but you are almost there! 🏃🏾 

## Highlights 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 

## Required Changes ♻️
It's ok to have bugs in your code. Please, see this code review as a chance to learn practical coding skills and new tips 🌟  that can improve your code quality 💯 and help you get a better job 💼.

I will guide 📔 you through fixing 🛠️ these bugs, follow my comments down below and make sure you don't miss any of them to get your project approved ✅

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/index.html.erb (hunk 1 above)
Comment (aron-net) [reply app/views/users/index.html.erb]: Already implemented it! Thanks for the feedback! :smile:

File: app/views/posts/show.html.erb (hunk 2 above)
Comment (aron-net) [reply app/views/posts/show.html.erb]: Already implemented it! Thanks for the feedback! :smile:

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @aron-helu  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Made changes as requested by the previous reviewer 👍 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**